modules/test/playwright/tests/portal-workflow-kaleo-designer-web/assignments.spec.ts

```diff
@@ -74,3 +74,32 @@ test('cannot see Scripted Assignment option when script management configuration
 		await page.$(`#assignment-type option[value="scriptedAssignment"]`)
 	).toBeNull();
 });
+
+test('role name search box does not support regex strings', async ({
+	diagramViewPage,
+	nodePropertiesSidebarPage,
+	page,
+	processBuilderPage,
+}) => {
+	await processBuilderPage.goto();
+
+	await processBuilderPage.clickWorkflowDefinitionName(
+		workflowDefinitionName
+	);
+
+	await diagramViewPage.clickNode('review');
+
+	await nodePropertiesSidebarPage.editAssignmentButton.click();
+
+	await page.locator('input#role-name').first().fill('a');
+
+	expect(
+		await page.getByRole('menuitem').filter({hasText: 'Account'}).first()
+	).toBeVisible();
+
+	await page.locator('input#role-name').first().fill('a*');
+
+	expect(
+		await page.getByRole('menuitem').filter({hasText: 'Account'})
+	).not.toBeVisible();
```

Comment:
Hey @devpedrotavares, if the field was still supporting regex would we still find Account as a result?

Reply:
Hi @igor-franca , yes we would. The `a+` string would bring all roles containing the letter 'a' at least one time.